.github/workflows/verify.yaml

```diff
@@ -25,6 +19,26 @@ jobs:
           checkstyle_version: '9.0'
           reporter: 'github-check'
           filter_mode: 'nofilter'
+          properties_file: resources/edc-checkstyle-ignore-enhanced.properties
+
+      - name: Run Checkstyle with additional checks for changed lines
```

Comment:
Why we need these two checkstyle steps now ?

Reply:
One is strict and the other not. You would suggest to run only the strict one but filter changed files only? Would be an option

Reply:
Yes, that can be a good option to run only one strict check with changed files.

Reply:
Implemented